app/controllers/members_controller.rb

```
@@ -21,7 +21,7 @@ def create
 
   def destroy
     participant = Participant.friendly.find(params[:id])
-    participant.update(organizer_id: nil)
+    @organizer.participants.destroy(participant)
```

Let's follow policy of failing early. Please use `destroy!` instead of `destroy`.